src/main/java/stringcalculator/model/StringCalculator.java

```diff
@@ -0,0 +1,62 @@
+package stringcalculator.model;
+
+import stringcalculator.exception.exceptions.OperandArgumentFormatException;
+
+import java.util.Arrays;
+
+public class StringCalculator {
+
+    private static final String DEFAULT_SPLIT_REGEX = "[,|:]";
+    private static final String CUSTOM_SPLITTER_PREFIX = "//";
+    private static final String CUSTOM_SPLITTER_SUFFIX = "\n";
+    private static final String POSITIVE_NUMBER = "0123456789";
+    private static final String SPLITTER_MARK = "\\";
+
+    public int extractSumNumbers(final String input) {
+        validateIsNotNull(input);
+
+        if (input.contains(CUSTOM_SPLITTER_PREFIX) && input.contains(CUSTOM_SPLITTER_SUFFIX)) {
```

Comment:
```suggestion
        if (isCustomType(...)) {
```

요런식으로 메서드 분리해줘도 가독성 좋을 것 같네요!